scripts/assetName.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const asset = require('../asset/asset');
+
+console.log(`${asset.name}-${asset.version}-${process.platform}-${process.arch}.zip`);
```

Comment:
The asset version should be prefixed with a `v`.  Additionally, let's change the build file name to include the major version of node, e.g. `node-8` since it is possible to have an incompatibility between node version. And while it may not be a problem now, it will difficult later down the road.

```suggestion
console.log(`${asset.name}-v${asset.version}-node-8-${process.platform}-${process.arch}.zip`);
```

Reply:
Any objections to 

```javascript
console.log(`${asset.name}-v${asset.version}-node-${process.version}-${process.platform}-${process.arch}.zip`);
```

I'm afraid if I put `node-8` in there it will be wrong for like 6 months until we notice it.